fern/apis/fern-definition/definition/types.yml

```diff
@@ -1,223 +1,224 @@
-imports: 
+imports:
   commons: commons.yml
   encoding: encoding.yml
   validation: validation.yml
   examples: examples.yml
   source: source.yml
 
-types: 
-  TypeDeclarationSchema: 
+types:
+  TypeDeclarationSchema:
     discriminated: false
-    union: 
+    union:
       - string
       - ObjectSchema
       - EnumSchema
       - DiscriminatedUnionSchema
       - UndiscriminatedUnionSchema
       - AliasSchema
-  
-  # InlinedTypeDeclaration: 
+
+  # InlinedTypeDeclaration:
   #   discriminated: false
-  #   union: 
+  #   union:
   #     - InlinedObjectSchema
   #     - InlinedEnumSchema
   #     - InlinedDiscriminatedUnionSchema
   #     - InlinedUndiscriminatedUnionSchema
-  
-  # InlinedObjectSchema: 
+
+  # InlinedObjectSchema:
   #   extends: ObjectSchema
-  #   properties:   
+  #   properties:
   #     name: string
 
-  # InlinedEnumSchema: 
+  # InlinedEnumSchema:
   #   extends: EnumSchema
-  #   properties:   
+  #   properties:
   #     name: string
-  
-  # InlinedDiscriminatedUnionSchema: 
+
+  # InlinedDiscriminatedUnionSchema:
   #   extends: DiscriminatedUnionSchema
-  #   properties:   
-  #     name: string  
+  #   properties:
+  #     name: string
 
-  # InlinedUndiscriminatedUnionSchema: 
+  # InlinedUndiscriminatedUnionSchema:
   #   extends: UndiscriminatedUnionSchema
-  #   properties:   
-  #     name: string  
+  #   properties:
+  #     name: string
 
-  TypeReferenceSchema: 
+  TypeReferenceSchema:
     discriminated: false
-    union: 
+    union:
       - string
       - TypeReferenceDetailedSchema
-  
-  BaseTypeReferenceSchema: 
-    extends: 
+
+  BaseTypeReferenceSchema:
+    extends:
       - commons.WithDocsSchema
       - commons.WithAvailability
-    properties: 
+    properties:
       default: optional<unknown>
       encoding: optional<encoding.EncodingSchema>
       validation: optional<validation.ValidationSchema>
-  
-  TypeReferenceDetailedSchema: 
-    extends: 
+
+  TypeReferenceDetailedSchema:
+    extends:
       - BaseTypeReferenceSchema
       - commons.WithName
       - commons.WithAudiences
-    properties: 
+    properties:
       type: string
 
-  TypeReferenceDeclarationWithNameSchema: 
+  TypeReferenceDeclarationWithNameSchema:
     discriminated: false
-    union: 
+    union:
       - string
       - TypeReferenceDeclarationWithName
-  
+
   TypeReferenceDeclarationWithName:
-    extends: 
+    extends:
       - BaseTypeReferenceSchema
       - commons.WithName
       - commons.WithAudiences
-    properties: 
+    properties:
       type: string
-  
-  BaseTypeDeclarationSchema: 
-    extends: 
+      inline: optional<boolean>
```

Comment:
i assume this is the only thing that changed? nit: would be nice to remove the formatting changes for this PR